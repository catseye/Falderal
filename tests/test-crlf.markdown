Falderal Test: CRLF
-------------------

These Falderal tests aim to show that the implementation
can compare the result with the expected text, regardless
of end-of-line convention (LF or CRLF).

    -> Functionality "CRLF" is implemented by
    -> shell command "python3 crlf.py"

    -> Tests for functionality "CRLF"

Cat CRLFs.

    | my kitty, 'tis of thee
    | feline felicity
    | something something
    = my kitty, 'tis of thee
    = feline felicity
    = something something

No CRLFs in test report (Intentional fail.)

    | one
    | two
    = three
