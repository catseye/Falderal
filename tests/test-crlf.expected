FAILED  : 

No CRLFs in test report (Intentional fail.)

Location: test-crlf.markdown, line 24
Impl    : shell command "python crlf.py"
Body    : 
one
two
Expected: output:
three
Actual  : output:
one
two

--------------------------------
Total test runs: 2, failures: 1
--------------------------------
